assets/css/pagination.less

```
@@ -144,27 +143,37 @@
         top: 0;
     }
 
-    img {
+    > img {
         width: 100%;
         height: 100%;
         object-fit: cover;
         
         border-radius: inherit;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)